### PR TITLE
FloatToUInt Conversion Issue

### PR DESCRIPTION
**Pull request by gleonoliva**
Convert to unsigned integer converts negative floats to 0.

### CODE REVIEW

File: source/core/Math.cpp [hunk 1]
```diff
@@ -413,6 +413,8 @@ DECLARE_SCALE2X_INTN_HELPER(Single)
             _Param(1) = numeric_limits<DEST>::max(); \
         } else if (::isinf(src)) { \
             _Param(1) = src < 0 ? numeric_limits<DEST>::min() : numeric_limits<DEST>::max(); \
+        } else if (src < numeric_limits<DEST>::min()) { \
+            _Param(1) = numeric_limits<DEST>::min(); \
```
Comment (sanmut) [inline source/core/Math.cpp]: Do we need to test for src greater than DEST::max()?

Comment (sanmut) [inline source/core/Math.cpp]: Please add the .via file and .vtr too.

**Comment by sanmut** [comment]
Not a big deal. It is a good practice to to squash some of the commits if they are not providing great value as a separate commit. It is a good idea to have separate commits for different logical fixes, so forth.

File: README.md [hunk 2]
```diff
@@ -21,6 +21,11 @@ Requirements:
 Setup:
 - `npm install`
 
+Setup (Windows):
```
Comment (cglzaguilar) [inline README.md]: For next time include unrelated changes to the fix in a different pull request. That makes them easier to understand.

File: test-it/testList.json [hunk 3]
```diff
@@ -286,6 +286,7 @@
                 "TemplateBasics.via",
                 "TestVI_ArrayInsertPartial.via",
                 "TestVIArrayMaxAndMin.via",
+				"TestVIConversion.via",
```
Comment (cglzaguilar) [inline test-it/testList.json]: Fix tab, other than looks good to me.
